### PR TITLE
DM-28926: save the config schema as code

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: python/lsst/ts/ATDomeTrajectory/config_schema.py [hunk 1]
```diff
@@ -1,4 +1,31 @@
-$schema: http://json-schema.org/draft-07/schema#
+# This file is part of ts_ATDomeTrajectory.
+#
+# Developed for the LSST Telescope and Site Systems.
```
Comment (wvreeven) [inline python/lsst/ts/ATDomeTrajectory/config_schema.py]: Please update this and all other files to Vera Rubin Observatory.

**Review by wvreeven — APPROVED** [review]
One minor comment but otherwise looks good.

File: python/lsst/ts/ATDomeTrajectory/config_schema.py [hunk 2]
```diff
@@ -1,4 +1,31 @@
...
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+
+__all__ = ["CONFIG_SCHEMA"]
+
+import yaml
+
+CONFIG_SCHEMA = yaml.safe_load(
+    """$schema: http://json-schema.org/draft-07/schema#
 $id: https://github.com/lsst-ts/ts_ATDomeTrajectory/blob/master/schema/ATDomeTrajectory.yaml
```
Comment (wvreeven) [inline python/lsst/ts/ATDomeTrajectory/config_schema.py]: Please update the path to reflect the correct file.

**Review by wvreeven — APPROVED** [review]
Added another minor comment.

File: python/lsst/ts/ATDomeTrajectory/config_schema.py (hunk 1 above)
Comment (r-owen) [reply python/lsst/ts/ATDomeTrajectory/config_schema.py]: Good catch. Done.

**Pull request merged**